crates/relayer/src/chain/endpoint.rs

```diff
@@ -685,4 +685,13 @@ pub trait ChainEndpoint: Sized {
         &self,
         request: QueryIncentivizedPacketRequest,
     ) -> Result<QueryIncentivizedPacketResponse, Error>;
+
+    fn save_conn_tx_hash(
```

Comment:
here is ok, but considering further use of saving tx hashes, is there a possibility to use a more abstract method to save different kinds of txs hash?